src/game/option/option_graphics.c

```diff
@@ -410,13 +412,15 @@ void Option_Graphics(INVENTORY_ITEM *inv_item)
             if (g_Config.ui.text_scale > MIN_TEXT_SCALE) {
                 g_Config.ui.text_scale -= 0.1;
                 reset = TEXT_UI_TEXT_SCALE;
+                CLAMP(g_Config.ui.text_scale, MIN_TEXT_SCALE, MAX_TEXT_SCALE);
             }
             break;
 
         case TEXT_UI_BAR_SCALE:
             if (g_Config.ui.bar_scale > MIN_BAR_SCALE) {
                 g_Config.ui.bar_scale -= 0.1;
                 reset = TEXT_UI_BAR_SCALE;
+                CLAMP(g_Config.ui.bar_scale, MIN_BAR_SCALE, MAX_BAR_SCALE);
```

Comment:
Do we need the `if`s still?

Reply:
I guess not? I could also add clamps for the brightness.